sound/soc/sof/intel/hda-loader.c

```diff
@@ -372,3 +372,148 @@ int hda_dsp_cl_boot_firmware(struct snd_sof_dev *sdev)
 	dev_err(sdev->dev, "error: load fw failed err: %d\n", ret);
 	return ret;
 }
+
+// skl/kbl enable core and code loader DMA has some difference with apl/cnl
```

Comment:
Please always use C style comments /* comment */

Reply:
OK, will change that.